packages/core/src/components/Onboarding/index.tsx

```diff
@@ -36,6 +37,11 @@ export function Onboarding({
 
   const [page, setPage] = useState<number>(platform === 'web' ? 1 : 0)
 
+  useEffect(() => {
```

Comment:
A `useEffect` is not ideal here — it's generally suspicious using a `useEffect` to achieve a "dependency effect", e.g. like achieving a "if X happens, then do Y". This can have some weird side effects!

Instead, we can put this logic directly on line 83, where the `primaryColor` gets set, so whenever the `primaryColor` gets set, we also just set the name there! Let me know if that makes sense — awesome job!

Reply:
Ah, that makes sense. I will implement that now.